### DISCUSSION TITLE
autoremove-torrents script

### DISCUSSION THREAD

**Discussion by mduijm** [transmission/transmission#5548] [Q&A]
Hi,

I use this tool:
https://github.com/jerrymakesjelly/autoremove-torrents

However it seems that right now it is broken with 4.0.3. As I am not a developer and it seems the original developer is on silent, is there someone here that can give a hint?

I think this file:
https://github.com/jerrymakesjelly/autoremove-torrents/blob/master/autoremovetorrents/conditionparser.py

does the work towards Transmission, but as I said, I am not a developer ;)

any help appreciated, ok even to buy a beer for the one fixes it ;)

**Comment by ksurl** [comment]
you can set `script-torrent-done-seeding-enabled` in settings to run a script. it can run transmission-remote command to remove the torrent
```
#!/bin/sh

transmission-remote localhost -t $TR_TORRENT_ID -r
```

**Comment by killemov** [comment]
Check out https://forum.transmissionbt.com/viewtopic.php?t=10364 for inspiration.
